api/v1alpha1/gatekeeper_types.go

```diff
@@ -34,7 +34,7 @@ type GatekeeperSpec struct {
 	Image *ImageConfig `json:"image,omitempty"`
 	// +optional
 	Audit *AuditConfig `json:"audit,omitempty"`
-	// +optional
+	// +kubebuilder:default:=Enabled
```

Comment:
We should still leave the `+optional` marker to indicate that this field is optional to coincide with the `omitempty` json tag.